backend/src/test/java/ai/verta/modeldb/RepositoryTest.java

```diff
@@ -196,20 +201,16 @@ public void createDeleteRepositoryNegativeTest() {
                     RepositoryIdentification.newBuilder()
                         .setNamedId(
                             RepositoryNamedIdentification.newBuilder()
-                                .setWorkspaceName("test1verta_gmail_com")
-                                .setName(repo2)
+                                .setWorkspaceName(testUser1UserName)
+                                .setName(repo1)
                                 .build())
                         .build())
                 .setRepository(Repository.newBuilder().setName(repo1))
                 .build();
         versioningServiceBlockingStub.createRepository(setRepository);
         Assert.fail();
       } catch (StatusRuntimeException e) {
-        if (testConfig.hasAuth()) {
```

Comment:
why did you get rid of this check?

Reply:
Test class committed by mistake so removed from this PR